ledger/ledger.go

```diff
@@ -296,9 +297,12 @@ func openLedgerDB(dbPathPrefix string, dbMem bool, cfg config.Local, log logging
 	go func() {
 		var lerr error
 		switch cfg.StorageEngine {
+		case "pebbledb":
+			dir := dbPathPrefix + "/tracker.pebble"
+			trackerDBs, lerr = pebbledbdriver.Open(dir, dbMem, config.Consensus[protocol.ConsensusCurrentVersion], log)
```

Comment:
I have changes to our data-dir pathing that affects this slightly by determining the `trackerDBPrefix` and `blockDBPrefix` just prior to this `go func`.

#5614

I'm perfectly happy to handle the merge on my side if this goes through first, just letting you know.